cypress/e2e/po/pages/explorer/projects-namespaces.po.ts

```diff
@@ -29,7 +30,63 @@ export default class ProjectNamespacePagePo extends PagePo {
     return this.createProjectNamespaceButton().click();
   }
 
+  createProjectButton() {
+    return this.self().getId('masthead-create');
+  }
+
+  createProjectButtonClick(): Cypress.Chainable {
+    return this.createProjectButton().click();
+  }
+
   nsProject() {
     return this.self().getId('name-ns-description-project');
   }
+
+  name() {
+    return LabeledInputPo.bySelector(this.self(), '[data-testid="name-ns-description-name"]');
```

Comment:
This should be accessed via an instance of `cypress/e2e/po/components/name-ns-description.po.ts`